### ISSUE TITLE
Installing voila in an anaconda environment - idna package missing

### ISSUE THREAD

**Issue by enryH** [voila-dashboards/voila#720]
Platform: Win10

I installed voila in an anaconda environment without installing jupyterlab along side. I go the same error as described [here]https://groups.google.com/a/continuum.io/g/anaconda/c/fWyuHt1eoH4?pli=1) (I missed to save it...)

I had to install the `conda install idna` manually additionally in the environment to get it to work. 

I don't know why it was missing, but I wanted to document it briefly.

**Comment by maartenbreddels** [comment]
Did you create a new environment? If not, could you create a new environment, this might be an old problem not related to voila that is fixed

**Comment by enryH** [comment]
No, I try to have my jupyter eco-system in a base environment and link the execution environments using ipykernels only.

**Comment by PranjalSahu** [comment]
This issue occured to me in Ubuntu when I did conda update --all.
After restarting the jupyter notebook kept getting the message AttributeError: module 'idna' has no attribute 'IDNAError'.
conda install idna solved the problem for me too.
